canonical-web-and-design/maas-ui
# refactor(ui): use React Router v6 Link components

## Done

- Update React Router Links to the v6 components
- Update tests to support the v6 compatibility router.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the React client.
- Make sure you can navigate through the app via the header links and links within the content.

## Fixes

Fixes: #3958.
